holmes/plugins/toolsets/__init__.py

```diff
@@ -78,6 +78,7 @@ def load_python_toolsets(dal: Optional[SupabaseDal]) -> List[Toolset]:
         BashExecutorToolset(),
         MongoDBAtlasToolset(),
         RunbookToolset(),
+        AzureSQLToolset(),
```

Comment:
Trying to understand why it is here? it is just merge issue or was missed in the original AzureSQLToolset?